tests/adapters/switches/remote_test.py

```diff
@@ -548,6 +548,38 @@ def test_get_vlans(self):
         assert_that(vrrp_group2.ips, is_([IPAddress("3.3.3.1")]))
         assert_that(vrrp_group2.priority, is_(100))
 
+    def test_get_vlan_interfaces(self):
+        self.requests_mock.should_receive("get").once().with_args(
+                url=self.netman_url+'/switches/toto/vlans/1/interfaces',
+                headers=self.headers
+        ).and_return(
+            Reply(
+                content=open_fixture('get_switch_hostname_vlans_vlan_interfaces.json').read(),
+                status_code=200)
+        )
+
+        interfaces = self.switch.get_vlan_interfaces(1)
+
+        assert_that(interfaces, is_([3000, 3001, 3002]))
```

Comment:
Based on pull request #96 this is supposed to be a list of strings, the interfaces names, not a list of integers.